godot-ffi/src/global.rs

```diff
@@ -131,57 +103,51 @@ impl<T> Global<T> {
 
 // Encapsulate private fields.
 mod global_guard {
+    use super::*;
     use std::ops::{Deref, DerefMut};
-    use std::sync::MutexGuard;
-
-    use super::InitState;
 
     /// Guard that temporarily gives access to a `Global<T>`'s inner value.
     pub struct GlobalGuard<'a, T> {
-        // Safety invariant: Is `Initialized`.
```

Comment:
Why was this removed? This is still the safety invariant of this struct right? I think it's useful to keep this here.

Reply:
reworded 👍